dkpro-core-lancaster-asl/src/main/java/org/dkpro/core/lancaster/LancasterStemmer.java

```diff
@@ -55,12 +62,51 @@
     @ConfigurationParameter(name = PARAM_STRIP_PREFIXES, mandatory = false, defaultValue = "false")
     private boolean stripPrefix;
 
+    /**
+     * Specifies an URL that should resolve to a location from where to load custom rules. If the location starts with
+     * {@code classpath:} the location is interpreted as a classpath location, e.g. "classpath:my/path/to/the/rules".
+     * Otherwise it is tried as an URL, file and at last UIMA resource.
+     * @see ResourceUtils
+     */
+    public static final String PARAM_CUSTOM_RULES = "customRules";
```

Comment:
For this component, the rules serve as a model. Hence, we'd use PARAM_MODEL_LOCATION here:

```
	public static final String PARAM_MODEL_LOCATION = ComponentParameters.PARAM_MODEL_LOCATION;
	@ConfigurationParameter(name = PARAM_MODEL_LOCATION, mandatory = false)
	protected String modelLocation;
```